pattern-1/activemq-deployment.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: activemq
+  labels:
+    app: activemq
+    tier: prod
+spec:
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: activemq
+        tier: prod
+    spec:
+      containers:
+      - image: docker.wso2.com/activemq 
+        name: activemq
+        imagePullPolicy: Never
```

Comment:
We can change this imagePullPolicy to ifNotPresent. Users may configure docker registry and will not always use docker.wso2.com. We can use this policy in all the deployments.